apps/frontend/src/pages/project/libs/components/setup-analytics-modal/setup-analytics-modal.tsx

```diff
@@ -143,12 +158,46 @@ const SetupAnalyticsModal = ({
 							<span className={styles["list-item-title"]}>Node.js 20</span>: The
 							script requires Node.js 20 to be installed on your machine.
 						</li>
+						<li>
```

Comment:
I think we should also mention that for now only unix based systems are supported

Reply:
should it be in prerequisites or in installation steps title in brackets 
![image](https://github.com/user-attachments/assets/65da39e5-322a-4f0b-a18c-8cd6e4f51b6b)
![image](https://github.com/user-attachments/assets/53073a56-42f8-4186-ae53-997adf804de5)

Reply:
Looks good to me now